data.js

```diff
@@ -1,4 +1,121 @@
+  },
+  {
+    id: 191,
+    type: "article",
+    title: "Personal website SEO setup",
+    category: ["web-dev"],
+    tech: ["JavaScript", "Google Analytics", "HTML", "CSS"],
+    datePublished: "January 6, 2021",
+    projectURL: "https://www.codementor.io/projects/web/create-a-contact-form-b2n9ltrdy1"
+  },
+  {
+    id: 190,
+    type: "article",
+    title: "Simple web contact form",
+    category: ["web-dev"],
+    tech: ["PHP", "HTML", "CSS"],
+    datePublished: "January 6, 2021",
+    projectURL: "https://www.codementor.io/projects/web/create-a-contact-form-b2n9ltrdy1"
+  },
+  {
+    id: 189,
+    type: "article",
+    title: "Plagiarism checker website",
+    category: ["web-dev"],
+    tech: ["Python", "Bootstrap"],
+    datePublished: "December 20, 2020",
+    projectURL: "https://www.codementor.io/projects/web/plagiarism-checker-website-atx32nf0oa"
+  },
+  {
+    id: 188,
+    type: "article",
+    title: "Simple web contact form",
+    category: ["web-dev"],
+    tech: ["PHP", "HTML", "CSS"],
+    datePublished: "January 6, 2021",
+    projectURL: "https://www.codementor.io/projects/web/create-a-contact-form-b2n9ltrdy1"
+  },
+  {
+    id: 187,
+    type: "article",
+    title: "Build a custom Google Maps theme",
+    category: ["web-dev"],
+    tech: ["JavaScript", "HTML", "CSS"],
+    datePublished: "February 24, 2021",
+    projectURL: "https://www.codementor.io/projects/web/build-a-custom-google-maps-theme-bf8levr6eg"
+  },
+  {
+    id: 186,
+    type: "article",
+    title: "Tinder style swipe mobile app",
+    category: ["mob-dev"],
+    tech: ["Kotlin", "Java", "Swift"],
+    datePublished: "April 09, 2021",
+    projectURL: "https://www.codementor.io/projects/mobile/tinder-style-swipe-mechanism-for-an-ios-app-bh9ph8n1co"
+  },
+  {
+    id: 185,
+    type: "article",
+    title: "Cryptocurrency price listing mobile app",
+    category: ["mob-dev"],
+    tech: ["React Native", "Swift", "Flutter", "Dart"],
+    datePublished: "December 01, 2021",
+    projectURL: "https://www.codementor.io/projects/mobile/cryptocurrency-price-listing-mobile-app-atx32meo88"
+  },
+  {
+    id: 184,
+    type: "article",
+    title: "Restaurants social mobile app",
+    category: ["mob-dev"],
+    tech: ["React Native", "Swift", "Flutter", "Dart"],
+    datePublished: "December 01, 2021",
+    projectURL: "https://www.codementor.io/projects/mobile/restaurants-social-mobile-app-atx32rvbqo"
+  },
+  {
+    id: 183,
+    type: "article",
+    title: "Break time reminder mobile app",
+    category: ["mob-dev"],
+    tech: ["React Native", "Kotlin", "Java", "Swift"],
+    datePublished: "March 15, 2021",
+    projectURL: "https://www.codementor.io/projects/mobile/break-time-reminder-mobile-app-bjzm27c836"
+  },
+  {
+    id: 182,
+    type: "article",
+    title: "Invoicing and payment reminder mobile app",
+    category: ["mob-dev"],
+    tech: ["React", "Node.js", "Express", "MongoDB"],
```

Comment:
Just a little fix to maintain consistency with other project meta information.
 - "Node.js" to be replaced with "Node"